lib/smart_app_launch/smart_stu2_suite.rb

```diff
@@ -222,5 +223,7 @@ class SMARTSTU2Suite < Inferno::TestSuite
 
     group from: :smart_token_introspection
 
+    group from: :smart_backend_services
```

Comment:
I think this should go before the token introspection tests.

Reply:
Updated so backend services shows up before token introspection tests.